src/gc/env/gcenv.object.h

```diff
@@ -64,19 +66,31 @@ class MethodTable
         return m_componentSize;
     }
 
+    bool Collectible()
```

Comment:
Agreed - this definition mirrors the one in `methodtable.h`, though: https://github.com/dotnet/coreclr/blob/master/src/vm/methodtable.h#L1912 . We'd need to rename that one too.